modin/core/storage_formats/pandas/query_compiler.py

```diff
@@ -2533,12 +2533,24 @@ def _callable_func(self, func, axis, *args, **kwargs):
     groupby_sum = GroupByReduce.register("sum")
 
     def groupby_mean(self, by, axis, groupby_kwargs, agg_args, agg_kwargs, drop=False):
+        by_columns = by
```

Comment:
Can we reuse any of the logic in `groupby_agg`, which maybe is doing something similar? https://github.com/modin-project/modin/blob/08ec2e1397cf48fd26cb2a69fb378bb2f38c5d16/modin/core/storage_formats/pandas/query_compiler.py#L2809-L2827

If we can, we should write a common utility function.

Reply:
Done